en/7-iterate-improve-frequently.md

```diff
@@ -32,390 +15,170 @@ altLangPage: 7-effectuer-iterations-ameliorations-constamment
-
-**Digital Service Playbook (US):** We should use an incremental, fast-paced style of software development to reduce the risk of failure. We want to get working software into users' hands as early as possible to give the design and development team opportunities to adjust based on user feedback about the service. A critical capability is being able to automatically test and deploy the service so that new features can be added often and be put into production easily.
-
-**Digital Service Standard (AU):** Design and build the service using the service design and delivery process, taking an agile and user-centred approach.
-
-Designing services in a user-centred way means that the services you deliver will be easy to use and convenient for the people who need to use them, helping them to stay in the digital channel.
-
-Designing using agile methods allows you to be more proactive and respond easily to change, both in technology and government policy. Services should be improved frequently; they will be cheaper and more accountable to users.
-
-**3. Review and improve services continually (Think - Digital Design Playbook (ISED)):** The [TBS Policy on Service](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=27916) and the [TBS Management Accountability Framework (MAF)](http://www.tbs-sct.gc.ca/hgw-cgf/oversight-surveillance/maf-crg/index-eng.asp) require regular reviews of priority services in order to identify opportunities for service redesign." By reviewing services regularly, service providers can ensure that services continue to be relevant, effective and efficient, and deliver best value to clients.
-
-Use quantitative and qualitative data to help with regular reviews of your service. User needs and technology evolve. There will always be room for improvement.
-
-**3. Apply agile principles and be iterative. (Do - Digital Design Playbook (ISED)):** Applying agile involves continuous improvement by analyzing data, making adjustments and iterating to identify and test improvements. Core principles of agile include defining a measurable goal; sharing ownership of the problem; taking small steps with visible impact; validating ideas/solutions with the target audience (e.g., the client); measuring success; reflecting, adjusting and iterating.
-
-Start with a representation or prototype of the solution that will be tested and revised based on feedback and insights. Each iteration improves on the previous version. Your understanding of a problem and how to address it evolves each time you refine an idea and re-craft potential solutions. Taking an iterative approach also helps you reduce risks. It makes big failures less likely and turns mistakes into learning opportunities.
+- are prototyped quickly, and enhanced with more features after the service goes live
+- can be changed quickly if, for example, government policy or technology changes
+- keep improving based on continual user feedback **(Digital Service Standard (Ontario))**
```

Comment:
Is this new content sourced from the Ontario Digital Service Standard? If not, you should remove the reference.